src/analysis/retail/priest/holy/modules/talents/BottomRow/Lightweaver.tsx

```diff
@@ -9,7 +9,7 @@ import ItemHealingDone from 'parser/ui/ItemHealingDone';
 import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
 import { SpellLink } from 'interface';
 import { calculateEffectiveHealing, calculateOverhealing } from 'parser/core/EventCalculateLib';
-import { isCastBuffedByLightweaver, getHeal } from '../../../normalizers/CastLinkNormalizer';
```

Comment:
Can you clean this function up and the associated linking logic since its no longer needed